ux.symfony.com/assets/styles/demos/live-memory.min.css

```diff
@@ -5,6 +5,7 @@
 @import "live-memory/app/reset";
 @import "live-memory/app/root";
 
+@import "../components/Banner";
```

Comment:
Not exactly in the scope of this PR, but why is this file suffixed with `.min.css`? My first thought was this is a built asset that shouldn't be commited.

Reply:
It's due to the CDN that mess with CSS files (using a compression that does not understand some new CSS syntaxes).. the only way we found to avoid that was to add the ".min." part in the file name...

Reply:
Would you mind throwing a comment at the top of the file explaining this. Future me will thank you!

Reply:
Absolutely, should have done it before ! 

Done